src/tactic/ui/filter/filter_element_wdg.py

```diff
@@ -361,6 +361,11 @@ def get_display(my):
             div.add(HiddenWdg("op", "is on"))
         elif op == '~':
             div.add("&nbsp;&nbsp;&nbsp;contains&nbsp;&nbsp;&nbsp;")
+        elif op == 'is':
+            #TODO: have this style apply to everything else and get rid of &nbsps
+            div.add("<div style='margin-top: 8px; margin-right: 15px;'>is</div>")
```

Comment:
Let's use a real DivWdg here like this
op_div= DivWdg("is")
op_div.add_styles("margin-top: 8px; margin-right: 15px")